src/services/web3/contracts/reader-arbitrum/contract.json

```diff
@@ -0,0 +1,79 @@
+{
+  "address": "0xBA491F95892244cc8B224611Dfd696aC8693c54F",
```

Comment:
Use `0xD5E42f45CC4ef59dE83999FAAd1bef25A7abf657`, and copy ABI from https://arbiscan.io/address/0xd5e42f45cc4ef59de83999faad1bef25a7abf657#code